dannce/interface.py

```diff
@@ -1169,8 +1154,7 @@ def dannce_train(params: Dict):
                 params["chan_num"] + params["depth"],
                 params["n_channels_out"],
                 3 if cam3_train else len(camnames[0]),
-                batch_norm=False,
-                instance_norm=True,
+                params["norm_method"],
```

Comment:
```suggestion
                norm_method=params["norm_method"],
```

@data-hound look good?

Reply:
Yes, this should do